public/app/plugins/datasource/grafana-azure-monitor-datasource/components/QueryEditor/QueryEditor.tsx

```diff
@@ -26,22 +27,23 @@ const QueryEditor: React.FC<BaseQueryEditorProps> = ({ query, datasource, onChan
     <div data-testid="azure-monitor-query-editor">
       <QueryTypeField query={query} onQueryChange={onChange} />
 
-      <VerticalGroup>
-        <EditorForQueryType
-          subscriptionId={subscriptionId}
-          query={query}
-          datasource={datasource}
-          onChange={onChange}
-          variableOptionGroup={variableOptionGroup}
-          setError={setError}
-        />
+      <EditorForQueryType
+        subscriptionId={subscriptionId}
+        query={query}
+        datasource={datasource}
+        onChange={onChange}
+        variableOptionGroup={variableOptionGroup}
+        setError={setError}
+      />
 
-        {errorMessage && (
+      {errorMessage && (
+        <>
+          <br />
           <Alert severity="error" title="An error occurred while requesting metadata from Azure Monitor">
             {errorMessage}
           </Alert>
-        )}
-      </VerticalGroup>
+        </>
+      )}
```

Comment:
I've got to fix this up.

We don't want to use Vertical Group because it enforces some funky flexbox stuff, but we do still want the space above the error message

Reply:
https://github.com/grafana/grafana/pull/33724 :)